test/browser/run-test.sh

```diff
@@ -87,23 +87,24 @@ if [ -n "$test_basic" ]; then
     # Don't run TestPages, TestPackages, and TestTerminal at all -- not testing external APIs
     TESTS="$TESTS
         TestAccounts
-        TestBonding
-        TestBridge
         TestFirewall
         TestKdump
         TestJournal
         TestLogin
-        TestNetworking
         TestServices
         TestSOS
         TestSystemInfo
-        TestTeam
         "
 
     # HACK: python3-linux-procfs can't be installed on rawhide
+    # HACK: NetworkManager 1.39.8 IpConfig.Addresses is an empty list
```

Comment:
Seems dangerous just to skip it, if it is not reported as it will trickle down into rest of Fedora soon